keyboards/clawsome/coupe/config.h

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2020 AAClawson (AlisGraveNil)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
```

Comment:
It's only a matter of time until someone decides to create a via keymap for these boards.  
At that point they will find a unique VID for you.  
If you'd rather choose one for yourself, now is the time. :)

Reply:
I love VIA, but I have no idea how to choose a VID for myself

Reply:
The usual convention is to use two ASCII characters, eg. the VID for the 1upkeyboards boards is `0x6F75` for `ou`. You can check https://yanfali.github.io/qmk_usb_usage/ to see which values are taken.

Reply:
added VID (0x7767)